library/src/main/java/com/meetme/support/fonts/FontManager.java

```diff
@@ -43,10 +43,8 @@
     private static final FontManagerImpl IMPL;
 
     static {
-        // TODO: SDK_INT == VERSION_CODES.N
-        if (Build.VERSION.SDK_INT == Build.VERSION_CODES.M && Build.VERSION.PREVIEW_SDK_INT > 0) {
+        if (Build.VERSION.SDK_INT >= 24) {
```

Comment:
Also, IIRC, there were differences even within the Nougat preview releases.  It's been a while, but I thought I had to make changes as newer releases were coming out.

Granted, those were previews, and ideally should not be shipped on production devices, so arguably not worth digging too far into.